ros/scripts/pose_visualiser

```diff
@@ -54,9 +55,13 @@ class PoseVisualiser(object):
         self.pose_text_publisher.publish(marker_array_text)
 
     def load_poses(self):
```

Comment:
This function should actually be replaced by the generic YAML loading function in `mas_tools`: https://github.com/b-it-bots/mas_tools/blob/master/src/mas_tools/file_utils.py